docs/section12/Rule12-9.md

```diff
@@ -0,0 +1,40 @@
+
+# Receptacle - Rule 12-9
+
+**Rule ID:** 12-9  
+**Rule Description:** User RMR Receptacle Schedule in Baseline RMR?  
```

Comment:
I'm not sure what this rule is trying to do that is not part of 12-8, could you elaborate?

Reply:
Based on the TCD spreadsheet examples, Rule 12-9 also has a check Building.compliance_path for code_compliant and beyond_code. Looks like if the building goes for beyond_code and the schedules are different in User and Baseline, a warning is raised. But I didn't include this in the file. 

But we can still combine 12-9 and 12-8? Maybe just add a building level check for the compliance path. Not sure how taking code compliance vs beyond code affects the schedule. Shouldn't both follow Rule 12-8?

Reply:
RCT can flag mismatched schedules and show a message that this is not an error if modeling is done for beyond code program.

Reply:
Do we need to check Building.compliance_path for Rule 12-8 as well? So Rule 12-8 is applicable when the building goes for code compliance and Rule 12-9 is applicable when the building goes for beyond code? Or do we want to combine these two rules?

Reply:
I think the two rules should stay separate and yes, it should be only for beyond code in 12-8 also.